File: day.c
#include <stdio.h>

int main(void) {
    FILE *input = fopen("input.txt", "r");
    char line[1024];
    /* discard first line */
    fgets(line, 1024, input);

    int product = 1;
    int a = 0, b = 0, c = 0;
    while (fgets(line, 1024, input)) {
        sscanf(line, "%d, %d, %d", &a, &b, &c);
        product *= (a+b+c);
    }
    printf("%d\n", product);
    fclose(input);
    return 0;
}

File: day.java
import java.io.*;
import java.util.Scanner;

public class day {
    public static void main(String args[]) {
        int product = 1;
        try {
            File inputfile = new File("input.txt");
            Scanner sc = new Scanner(inputfile);
            sc.nextLine(); // discard first line
            while (sc.hasNextLine()) {
                int sum = 0;
                String[] valuesAsStrings = sc.nextLine().split(",", -1);
                for (String value : valuesAsStrings) {
                    sum += Integer.parseInt(value.trim());
                }
                product *= sum;
            }
            System.out.println(product);
        } catch(Exception e) {
            e.printStackTrace();
        }
    }
}
